$NetBSD: patch-aa,v 1.2 2009/01/09 22:22:04 markd Exp $

CMAKE_MODULE_PATH may be a path (ie multiple directories) so don't use it.

--- CMakeLists.txt.orig	2009-01-03 23:06:22.000000000 +1300
+++ CMakeLists.txt
@@ -225,7 +225,7 @@ if (CppUnit_FOUND)
 endif (CppUnit_FOUND)
 
 configure_file(
-  "${CMAKE_MODULE_PATH}/cmake_uninstall.cmake.in"
+  "${CMAKE_CURRENT_SOURCE_DIR}/cmake/cmake_uninstall.cmake.in"
   "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
   IMMEDIATE @ONLY)
 
